static/app/components/devtoolbar/components/alerts/alertsPanel.tsx

```diff
@@ -154,22 +149,18 @@ function AlertListItem({item}: {item: Incident}) {
         <TimeSince date={item.dateStarted} unitStyle="extraShort" />
       </div>
 
-      <TextOverflow css={smallCss} style={{gridArea: 'name'}}>
-        <SentryAppLink
-          to={{
-            url: alertDetailsLink({slug: organizationSlug} as Organization, item),
-            query: {alert: item.identifier},
-          }}
-          onClick={() => {
-            trackAnalytics?.({
-              eventKey: `devtoolbar.alert-list.item.click`,
-              eventName: `devtoolbar: Click alert-list item`,
-            });
-          }}
-        >
-          <strong>{item.title}</strong>
-        </SentryAppLink>
-      </TextOverflow>
+      <AnalyticsProvider nameVal="item" keyVal="item">
```

Comment:
<img width="497" alt="SCR-20240730-mjhv" src="https://github.com/user-attachments/assets/2c73466c-9864-4e40-a4e0-06239714fb39">

Reply:
added to amplitude